src/library/dao/directorydao.cpp

```diff
@@ -186,18 +190,26 @@ QList<RelocatedTrack> DirectoryDAO::relocateDirectory(
         loc_ids.append(DbId(query.value(1).toInt()));
         auto trackId = TrackId(query.value(0));
         auto oldLocation = query.value(2).toString();
+        if (!oldLocation.startsWith(oldLocationPrefix)) {
+            // LIKE is case-insensitive and may result in false positives
```

Comment:
This applies only to case sensitive file systems/OSs 
Do we have a chance tomake this conditional?

Reply:
I don't know and do not plan to change this. The locations in the database should be case-correct.